src/main/java/UC/WeatherRunner/WeatherRunnerController.java

```diff
@@ -0,0 +1,12 @@
+package UC.WeatherRunner;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+
```

Comment:
The application was without a controller. I set up the shell of one with /home as the route